packages/ibm-products/src/components/FilterPanel/FilterPanelLabel/FilterPanelLabel.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { getDevtoolsProps } from '../../../global/js/utils/devtools';
+import { pkg } from '../../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--filter-panel-label`;
+const componentName = 'FilterPanelLabel';
+
+const defaults = {
+  setCountAriaLabel: (count) => `${count} items`,
+};
+
+/**
+ * A container with a label and optional count.
+ */
+export let FilterPanelLabel = React.forwardRef(
+  (
+    {
+      children: label,
+      className,
+      count,
+      setCountAriaLabel = defaults.setCountAriaLabel,
+      labelTitle,
+      ...rest
+    },
+    ref
+  ) => {
+    return (
+      <span
+        {...rest}
+        className={cx(blockClass, className)}
+        ref={ref}
+        {...getDevtoolsProps(componentName)}
+      >
+        <span className={`${blockClass}__text`} title={labelTitle}>
```

Comment:
title shouldn't be necessary here

Reply:
The filter panel (and the labels) have very limited horizontal space. It's easy for database-driven labels to be long enough to get cut off. A `title` would at least allow a user to hover over the label and see the full text.

Reply:
Also, we are passing the label text as `children`, which allows the dev to use `React.node`. Should we switch to label as a proper prop and limit it to "string"?

Reply:
>The filter panel (and the labels) have very limited horizontal space. It's easy for database-driven labels to be long enough to get cut off. A title would at least allow a user to hover over the label and see the full text.

This has been a theme that’s come up in discussion with @kristastarr on the Datagrid reviews too. I think we can keep as is for this first PR even though I don’t think this is preferred behavior based on what I’ve started reading.

Since it applies to Datagrid as well, I think we should open this as further issue/discussion.

Reply:
All good points. 

On further research, I found the following:
 - `FilterPanelLabel` is used primarily by the checkbox, group, and accordion subcomponents.
 - `FilterPanelCheckbox` specifically returns Carbon's `Checkbox` as-is.

I thought I would standardize on how Carbon's `Checkbox` does it, since we're using it's "label" feature anyways, using the following props:
 - `labelText` is a `node` (required, allows any string or markup)
 - `title` is a `string` (optional, string only, because any markup will show as [Object object] in the tooltip)